.gitignore

```diff
@@ -46,7 +46,7 @@ dlldata.c
 project.lock.json
 project.fragment.lock.json
 artifacts/
-**/Properties/launchSettings.json
+# **/Properties/launchSettings.json
```

Comment:
Is this intentionally commented out?

Reply:
@matthchr - Thanks for reviewing. Yes that is intentional because I use that launchSettings.json to set a working directory for the app that works equally well under VS or .NET core. At least, this is what worked for me. I will delete the line from .gitignore.